README.md

```diff
@@ -15,4 +16,28 @@ markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
 
+## Definition of little $o$ 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+## Definition of Big $O$
+$f(n)\in O(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) \le c g(n)$
+
+## Proof 
+- If we start with $f(n)\in o(g(n))$ by the deffiniton provided we know that for any positive constant $c$ there exists a point called 
+  $n_0$ beyond which is always $\ f(n) < c \cdot g(n)$
+- If we were to choose another constant $k$ that is greater then 0.
+- We can then use the new constant for the definition of big $O$ to show that $f(n)\in O(g(n))$
+- Since we know that $\ \forall n\ge n_0: f(n) < c g(n)$
+- We can see that $\ f(n) \le k g(n)$ using the new constant $k$.
+- since we used a constant $k$ that is in the range of $c$ we have shown that the definition of little$o$ and Big $O$ is satisfied.
```

Comment:
What do you mean by "the range of $c$"?

Reply:
By this I meant that k > 0. As given in the definition of both big O and little o for c.

Reply:
So the value of $c$ is irrelevant?

Reply:
Yes kind of. By looking at the definitions it would seem that the c would take part in determining on the of n_0 where the function F(n) <= c(g(n)) . So the c does not exactly matter because based of the definitions there is a point n_0 where all n greater then n_0, f(n) <= c(g(n))

Reply:
Kind of? Not sure what you mean...

Reply:
I guess I am saying no the c is not relevant.

Reply:
I was trying to explain that I think c would have something to do with determining the point n_0 where n >n_0 makes all F(n) < c(g(n)). This would be the same case for the definition of Big O.

Reply:
In principle yes, but is this relevant here?

Reply:
No it is not relevant.

Reply:
So then please update your proof accordingly.